tests/unit/components/shared/FlaggedFeature.spec.js

```diff
@@ -1,30 +1,25 @@
-import { localVue, mount } from '@support/testHelper'
-
+import { mount } from '@support/testHelper'
 import FlaggedFeature from '@/components/shared/FlaggedFeature'
-import { vi } from 'vitest'
 
 describe('FlaggedFeature.vue', () => {
   beforeEach(() => {
-    // Ideally I'd love to mock the http response here, but swrv seems to tun
-    // into problems mounting via-vue test utils, and `getCurrentInstance` fails
-    // to find the instance
-    vi.mock('swrv', () => ({
-      default: vi.fn(() => ({
-        data: {
-          flipper_id: 'User',
-          features: {
-            enable_feature: { enabled: true },
-            disabled_feature: { enabled: false },
-          },
-        },
-      })),
-    }))
+    // Temporary workaround it has been moved to global mock in __mocks__
```

Comment:
can this be removed?

Reply:
This used a bit of a work around by moving the features to globals when they should be just for this test file. I will look into reverting this.